File: Split_the_Array.java
class Solution {
    public boolean isPossibleToSplit(int[] nums) {
        HashMap<Integer, Integer> count = new HashMap<>();

        for (int n : nums) {
            count.put(n, count.getOrDefault(n, 0) + 1);
        }
        
        for (int val : count.values()) {
            if (val > 2) {
                return false;
            }
        }
        
        return true;        
    }
}

File: Split_the_Array.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var isPossibleToSplit = function(nums) {
    let count = new Map();

    for (let n of nums) {
        count.set(n, (count.get(n) || 0) + 1);
    }
    
    for (let val of count.values()) {
        if (val > 2) {
            return false;
        }
    }
    
    return true;    
};
